.github/DEVELOPMENT.md

```diff
@@ -26,22 +26,24 @@ A typical pull request should strive to contain a single logical change (but not
 necessarily a single commit). Unrelated changes should generally be extracted
 into their own PRs.
 
-If a pull request does consist of multiple commits, it is expected that every
-prefix of it is correct. That is, there might be preparatory commits at the 
-bottom of the stack that don't bring any value by themselves, but none of the
-commits should introduce an error that is fixed by some future commit. Every
-commit should build and pass all tests.
-
-Commit messages and history are also important, as they are used by other
-developers to keep track of the motivation behind changes. Keep logical diffs
-grouped together in separate commits, and order commits in a way that explains
-the progress of the changes. Rewriting and reordering commits may be a necessary
-part of the PR review process as the code changes. Mechanical changes (like
-refactoring and renaming)should be separated from logical and functional
-changes. E.g. deduplicating code or extracting helper methods should happen in a
-separate commit from the commit where new features or behavior is introduced.
-This makes reviewing the code much easier and reduces the chance of introducing
-unintended changes in behavior.
+If a pull request does consist of multiple commits, it is expected that
+every prefix of it is correct. That is, there might be preparatory
+commits at the bottom of the stack that don't bring any value by
+themselves, but none of the commits should introduce an error that is
+fixed by some future commit. Every commit should build and pass all
+tests.
+
+Commit messages and history are also important, as they are used by
+other developers to keep track of the motivation behind changes. Keep
+logical diffs grouped together in separate commits, and order commits
+in a way that explains the progress of the changes. Rewriting and
+reordering commits may be a necessary part of the PR review process as
+the code changes. Mechanical changes (like refactoring and
+renaming)should be separated from logical and functional changes. E.g.
+deduplicating code or extracting helper methods should happen in a
+separate commit from the commit where new features or behavior is
+introduced.  This makes reviewing the code much easier and reduces the
+chance of introducing unintended changes in behavior.
```

Comment:
Why? It seems inconsistent with the rest of the document. Also, 1970 called and they want their dumb terminal back :)

Reply:
And we should use at least 120 in all files

Reply:
I'm sorry. I'm trying to learn on the go from comments on the PR. The other day I got this one: https://github.com/trinodb/trino/pull/19191#issuecomment-1745402569 from @electrum. Linked document mentions to use 72 chars column width for git commit messages and description. I 'extrapolated' this to README files. If I was wrong and there is some other number I should be using, I'm happy to hear.

Reply:
That comment is only about commit message titles, which indeed should be as short as possible (because otherwise they can get truncated by some tools, including GitHub). And maybe commit messages themselves. But there's no need for this in text documents.

BTW, I'm not requesting this to be reverted, I was just curious why.